### PR TITLE
WIP [DOCS-6176] add and get security marks]

### PR DESCRIPTION
**Pull request by oliveralfresco2018**
I have written:

- Added Add/Get Security Marks
- Audit Events and Triggers
- Delete a released project from the Admin App
- Save as Function

### CODE REVIEW

**Review by rallegre — APPROVED** [review]
Ok good for me

File: process-automation/latest/model/connectors/content.md [hunk 1]
```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
...
+
+The **ADD_SECURITY_MARKS** action is used to add security marks to specific content.  
+To add security marks in Process Automation you must have configured security marks in Governance Services. For information on security marks in Governance Services, see [Security Marks and Classification
+]({% link governance-services/latest/using/smc.md %})
+
+The input parameters from add security marks are:
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| file | file | *Required.* ACS file to be updated. File variables are initialised by content actions or content events in triggers. E.g: 'Initialise a file variable by mapping it to the output of the generate document task. |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/content.md:610]: When you refer to Content Services here you use ACS: "Required. ACS file to be updated." , should we stick to Alfresco Content Services as we are talking about another product, or I see some use of Content Services. But I think ACS could be confusing at first if you are not familiar with the abbreviation. 
This goes for all ACS usage below...

Comment (gravitonian) [reply process-automation/latest/model/connectors/content.md:610]: Change "the generate document" to "the generated document"

File: process-automation/latest/model/connectors/content.md [hunk 2]
```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
...
+| folder | folder | *Required.* ACS folder to be updated. Folder variables are initialised by content actions or content events in triggers. |
+| path | String | *Required.* Location or relative path of the node to be updated. For example a location path could be `/app:company_home/app:user_homes/cm:hruser` and a relative path could be `/User Homes/hruser` (by default the relative path is from Company Home). |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/content.md:612]: Change "updated. For example a location" to "updated. For example, a location"

File: process-automation/latest/model/connectors/content.md [hunk 3]
```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
...
+| nodeId | String | *Required.* Node ID of the file or folder in ACS. Example:\n'a6a977a6-c728-4038-8dbc-d914c4d8cfb3. |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/content.md:613]: \n does not work as line break. Use `<br>`

File: process-automation/latest/model/connectors/content.md [hunk 4]
```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
...
+The **GET_SECURITY_MARKS** action is used to get security marks from specific content.
+
+The input parameters from get security marks are:
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| file | file | *Required.* ACS file from which security maks are retrieved. File variables are initialised by content actions or content events in triggers. E.g: 'Initialise a file variable by mapping it to the output of the generate document task. |
+| folder | folder | *Required.* ACS folder from which security maks are retrieved. Folder variables are initialised by content actions or content events in triggers. |
+| path | String | *Required.* Location or relative path of the node in ACS. For example a location path could be `/app:company_home/app:user_homes/cm:hruser` and a relative path could be `/User Homes/hruser` (by default the relative path is from Company Home). |
+| nodeId | String | *Required.* Node ID of the file or folder in ACS. Example:\n'a6a977a6-c728-4038-8dbc-d914c4d8cfb3 |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/content.md:632]: \n does not work as line break. Use `<br>`

File: process-automation/latest/model/connectors/content.md [hunk 5]
```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
...
+
+The **GET_SECURITY_MARKS** action is used to get security marks from specific content.
+
+The input parameters from get security marks are:
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| file | file | *Required.* ACS file from which security maks are retrieved. File variables are initialised by content actions or content events in triggers. E.g: 'Initialise a file variable by mapping it to the output of the generate document task. |
+| folder | folder | *Required.* ACS folder from which security maks are retrieved. Folder variables are initialised by content actions or content events in triggers. |
+| path | String | *Required.* Location or relative path of the node in ACS. For example a location path could be `/app:company_home/app:user_homes/cm:hruser` and a relative path could be `/User Homes/hruser` (by default the relative path is from Company Home). |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/content.md:631]: Change "in ACS. For example a location" to "in ACS. For example, a location"

File: process-automation/latest/model/connectors/content.md [hunk 6]
```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
...
+| path | String | *Required.* Location or relative path of the node to be updated. For example a location path could be `/app:company_home/app:user_homes/cm:hruser` and a relative path could be `/User Homes/hruser` (by default the relative path is from Company Home). |
+| nodeId | String | *Required.* Node ID of the file or folder in ACS. Example:\n'a6a977a6-c728-4038-8dbc-d914c4d8cfb3. |
+| securityGroupName | String | *Required.* Security group that contains the security marks to be assigned. E.g: 'PII'. |
+| securityMarks | array | *Required.* Array including the name of the security marks to be added. |
+
+The output parameters from add security marks are:
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| response | json | *Optional.* Response for the calls. |
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/content.md:621]: No need to use *Optional* here as the developer cannot control output parameters.

File: process-automation/latest/model/connectors/content.md [hunk 7]
```diff
@@ -597,6 +597,48 @@ The output parameters from declaring a record are:
...
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| file | file | *Required.* ACS file from which security maks are retrieved. File variables are initialised by content actions or content events in triggers. E.g: 'Initialise a file variable by mapping it to the output of the generate document task. |
+| folder | folder | *Required.* ACS folder from which security maks are retrieved. Folder variables are initialised by content actions or content events in triggers. |
+| path | String | *Required.* Location or relative path of the node in ACS. For example a location path could be `/app:company_home/app:user_homes/cm:hruser` and a relative path could be `/User Homes/hruser` (by default the relative path is from Company Home). |
+| nodeId | String | *Required.* Node ID of the file or folder in ACS. Example:\n'a6a977a6-c728-4038-8dbc-d914c4d8cfb3 |
+| securityGroupName | String | *Required.* Security group that contains the security marks to be assigned. E.g: 'PII'. |
+
+The output parameters from get security marks are:
```
Comment (gravitonian) [inline process-automation/latest/model/connectors/content.md:635]: No need to use *Optional* here as the developer cannot control output parameters.

**Review by gravitonian — CHANGES_REQUESTED** [review]

File: process-automation/latest/model/connectors/content.md (hunk 7 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/content.md:635]: DOne

File: process-automation/latest/model/connectors/content.md (hunk 2 above)
Comment (oliveralfresco2018) [reply process-automation/latest/model/connectors/content.md:612]: Done

**Review by gravitonian — APPROVED** [review]

**Pull request merged**